book/02-git-basics/sections/recording-changes.asc

```diff
@@ -28,7 +28,7 @@ Your branch is up-to-date with 'origin/master'.
 nothing to commit, working directory clean
 ----
 
-This means you have a clean working directory – in other words, there are no tracked and modified files.
+This means you have a clean working directory – in other words, there are no modified files among tracked ones.
```

Comment:
I actually thought the _tracked_ is a mistake altogether and the sentence is supposed to be:

> This means you have a clean working directory – in other words, there are no staged and modified files.

Which I would change to:
> This means you have a clean working directory – in other words, there are no modified or staged files.

If the tracked is indeed supposed to be there, I would simplify the sentence even more:

> This means you have a clean working directory – in other words, no tracked files are modified.